### DISCUSSION TITLE
I am new to Github and can't download some software given the link I have been sent by the author

### DISCUSSION THREAD

**Discussion by Bradscopegems** [community/community#30953] [Other Feature Feedback, Questions, & Ideas]
I have worked with John Dempster, a code writer who is very famous in the world of electrophysiological research. He sent me , nearly two years ago, the following link, but I was unable to download it from Github because of family problems. Now I enter it and try to search, but his name or the whole link are both returned as producing zero hits. What am I doing wrong? The link he sent is 
https://github.com/johndempster/MesoscanV1/releases/tag/!.7.0  . The advanced search seems to be only for expert software writers who understand the jargon. I am one of the designers of the Mesolens, a new type of microscope lens which captures orders of magnitude more information than conventional microscope objectives.

**Comment by airtower-luna** [comment]
The link indeed seems to be broken, most likely the repository has been deleted or made private. I see there's one other repository with "mesoscan" in its name under the same account (https://github.com/johndempster/MesoScanXE), but considering you had previous contact your best option is probably to ask the author what happened.

**Comment by tenmark86** [comment]
@Bradscopegems The link is okay, am able to access the repo https://github.com/johndempster/MesoScanXE 

![image_2022-08-29_224616688](https://user-images.githubusercontent.com/83606182/187285263-e6e17c31-ba34-4ef5-8ab6-6f9d58b836bb.png)
